src/components/Search/SearchOutputTable.vue

```diff
@@ -1,21 +1,18 @@
 <template>
-    <table class="table">
+    <table class="table table-hover">
         <thead>
             <tr>
-                <th>Name</th>
-                <th>Type</th>
-                <th>Subject</th>
-                <th>...</th>
+                <th v-for="(header, name, index) in headers" :key="index">
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [':key' should be on a new line.](https://app.codacy.com/gh/FAIRsharing/fairsharing.github.io/pullRequest?prid=4893952)